index.ios.js

```diff
@@ -75,14 +72,15 @@ var WalkAbout = React.createClass({
     }
   },
 
-  toggleGeofenceState: function() {
-    this.state.inGeofence = !this.state.inGeofence;
+  backToMap: function(){
+    this.setState({inGeofence: false})
   },
 
   render: function() {
     if(this.state.inGeofence){
       console.log('rendering MonumentDetail')
-      return (<MonumentDetail monument={this.currentMonument} />)
+      return (<MonumentDetail monument={MONUMENTS[0]}
```

Comment:
Right now the only reason I'm not merging this is the fact that it's passing in MONUMENTS[0] instead of this.currentMonument. I want to go over this, but everything else looks fantastic.